src/postgraphql/withPostGraphQLContext.ts

```diff
@@ -163,7 +163,7 @@ async function setupPgClientTransaction ({
   // this prevents an accidentional overwriting
   if (typeof pgSettings === 'object') {
     for (const key of Object.keys(pgSettings)) {
-      localSettings.set(key, String(pgSettings[key]))
+      localSettings.set(key, !!pgSettings[key] ? String(pgSettings[key]) : null)
```

Comment:
This is not a safe way to check if the value should be excluded - they might deliberately be setting it to the empty string `""` or to the number zero `0` and this will instead set `null`. A safer way would be `pgSettings[key] != null ? String(...) : null`; however I'd rather throw if the values are invalid:

> strings and numbers should be passed through, null/undefined should be dropped, and everything else should throw an error.
> -- https://github.com/postgraphql/postgraphql/issues/566#issuecomment-325129925

So I'd like to see some `typeof`'s around here. For anyone wondering: I'm deliberately suggesting we should throw on booleans, and on objects that have a `toString` value.